### PR TITLE
Adding method to relay using Certificate based inbound connector

### PR DESCRIPTION
**Pull request by arindamthokder**
Adding change for query on https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1792
FYI- I proposed a title change for https://docs.microsoft.com/en-us/exchange/troubleshoot/connectors/office-365-notice
Not sure of that would cause the URL also to change, if that happens, please include the latest URL in my change here.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mail-flow-best-practices/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-3.md [hunk 1]
```diff
@@ -334,6 +334,10 @@ Here's a comparison of each configuration option and the features they support.
 |**Limitations**|
 |Throttling limits|10,000 recipients per day. 30 messages per minute.|Standard throttling is in place to protect Office 365.|Reasonable limits are imposed. The service can't be used to send spam or bulk mail. For more information about reasonable limits, see [Higher Risk Delivery Pool for Outbound Messages](https://go.microsoft.com/fwlink/p/?linkid=830829).|
 
+##Configure a certificate-based connector to relay email through Office 365
+
+If your devices or applications are capable of using a certificate for mailflow, you can configure relay through Office 365 by verifying the subject name on the certificate that the sending device or application uses. The Common-Name (CN) or Subject Alternate Name (SAN) in the certificate should contains a domain name that you have registered in Office 365. Also, you have to create a certificate-based connector in Office 365 that has the same domain name mentioned to accept and relay emails coming from these devices, applications or any other On-premises server. For more details about this method please refer to- https://docs.microsoft.com/en-us/exchange/troubleshoot/connectors/office-365-notice
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-3.md:339]:
```suggestion
If your devices or applications are capable of using a certificate for mail flow, you can configure relay through Office 365 by verifying the subject name on the certificate that the sending device or application uses. The common name (CN) or subject alternative name (SAN) in the certificate should contain a domain name that you have registered in Office 365. Also, you have to create a certificate-based connector in Office 365 that has the same domain name mentioned to accept and relay emails coming from these devices, applications, or any other on-premises server. For more details about this method, please refer to this [important notice for Office 365 email customers who have configured connectors](https://docs.microsoft.com/exchange/troubleshoot/connectors/office-365-notice). 
```

**Comment by scanum** [comment]
@mattpennathe3rd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by mattpennathe3rd** [comment]
@arindamthokder Thanks very much for helping with this! I have incorporated this PR into another, so your additions will be published shortly after that PR is merged.

**Pull request closed without merging**